biztalk/technical-guides/how-to-enable-receive-location-fault-tolerance.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
since this is a new file, you also need to update the TOC file:
https://github.com/msnehagup/biztalk-docs/blob/pravagar/partialdrl/biztalk/TOC.md